lldb/test/API/lit.site.cfg.py.in

```diff
@@ -44,6 +44,10 @@ lldb_build_intel_pt = '@LLDB_BUILD_INTEL_PT@'
 if lldb_build_intel_pt == '1':
     config.enabled_plugins.append('intel-pt')
 
+llvm_enable_curl = '@LLVM_ENABLE_CURL@'
```

Comment:
Is this correct? Won't this expand to 'NO' which is truthy when expanded with it off? Maybe you want this without quotes?

(Looks ike LLDB_BUILD_INTEL_PT gets this wrong too.)

Reply:
I validated that this works correctly (had the same work). The *reason* it works is because I added LLVM_ENABLE_CURL to the list in `llvm_canonicalize_cmake_booleans` found in lldb/test/CMakeLists.txt.

Reply:
Emulating the curses dependency approach would probably be more consistent (grep for `skipIfCursesSupportMissing`). Intel-pt does something different because it lives outside of (lib)lldb.